ext/search_kit/tests/phpunit/api/v4/SearchDisplay/EntityDisplayTest.php

```diff
@@ -22,7 +22,17 @@ public function setUpHeadless(): CiviEnvBuilder {
       ->apply();
   }
 
-  public function testEntityDisplay() {
+  public static function getDataModes(): array {
+    return [
+      ['table'],
+      ['view'],
```

Comment:
Current test failures are likely because (1) the tests are running on MySQL 5.7 and (2) this new functionality requires MySQL 8.0.